src/main/java/frc/robot/subsystems/tiltedelevator/TiltedElevatorSubsystem.java

```diff
@@ -166,11 +166,11 @@ public void periodic() {
             // System.out.println(leftHallSensorPos);
             // System.out.println(lastHallPos + " " + leftHallSensorPos);
             if (leftHallSensorPos != null && lastHallPos == null){
-                extensionEncoder.setPosition(Units.inchesToMeters(EXTENSION_LIMIT));//leftHallSensorPos.getExtendDistanceMeters());
```

Comment:
I know this is commented out but `EXTENSION_LIMIT` is already in meters. Converting it as if it was in inches again would lead to an incorrect number; is that why it was commented?
```java
extensionEncoder.setPosition(leftHallSensorPos.getExtendDistanceMeters());
```
seems correct here.